addSite.js

```diff
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', function() {
+	document.getElementById('urlInput').addEventListener("keypress", function(event){
+		if(event.keyCode == 13){ // 13 is enter
+			addUrlToFilterList();
+		}
+	});
+    let addButton = document.getElementById('add');
+    addButton.addEventListener('click', function() {
+		addUrlToFilterList();
+    });
+});
```

Comment:
można te tabulatury ewentualnie powyrównywać

Reply:
That file is not necessary anymore. It should be already removed. Sry. :)